h2o-docs/src/product/data-science/algo-params/max_depth.rst

```diff
@@ -30,269 +30,269 @@ Related Parameters
+            stopping_tolerance = 1e-4)
+
+        # Build grid search with previously made GBM and hyper parameters
+        grid = H2OGridSearch(gbm_grid,hyper_params,
+                             grid_id = 'depth_grid',
+                             search_criteria = {'strategy': "Cartesian"})
+
+        # Train grid search
+        grid.train(x=predictors, 
+                   y=response,
+                   training_frame = train,
+                   validation_frame = valid)
+
+        # Display the grid search results
+        # Sorted by increasing logloss (because this is a classification task)
+        print grid
+
+             max_depth            model_ids              logloss
+        0           17   depth_grid_model_8  0.20544094075930078
+        1           19   depth_grid_model_9  0.20584402503242194
+        2           27  depth_grid_model_13  0.20627418156921704
+        3           11   depth_grid_model_5   0.2069364255413584
+        4           13   depth_grid_model_6   0.2078569528636169
+        5           25  depth_grid_model_12  0.20834760530631993
+        6            9   depth_grid_model_4  0.20842232867415922
+        7           29  depth_grid_model_14  0.20904163538087436
+        8           15   depth_grid_model_7  0.20991531457742935
+        9           23  depth_grid_model_11   0.2104361858121492
+        10          21  depth_grid_model_10  0.21069590143686837
+        11           7   depth_grid_model_3  0.21127939637392396
+        12           5   depth_grid_model_2  0.21509420086032935
+        13           3   depth_grid_model_1  0.21854010261642962
+        14           1   depth_grid_model_0  0.23892331983893703
+
+        # Sort the grid models by decreasing AUC
+        sorted_grid = grid.get_grid(sort_by='auc',decreasing=True)
+        print(sorted_grid)
+
+             max_depth            model_ids                 auc
+        0           13   depth_grid_model_6  0.9552831783601015
+        1           27  depth_grid_model_13  0.9547196393350239
+        2           17   depth_grid_model_8  0.9543251620174698
+        3           11   depth_grid_model_5  0.9538743307974078
+        4            9   depth_grid_model_4  0.9534798534798535
+        5           19   depth_grid_model_9  0.9534234995773457
+        6           25  depth_grid_model_12  0.9529726683572838
+        7           29  depth_grid_model_14  0.9528036066497605
+        8           21  depth_grid_model_10  0.9526908988447449
+        9           15   depth_grid_model_7  0.9526345449422373
+        10           7   depth_grid_model_3   0.951789236404621
+        11          23  depth_grid_model_11  0.9505494505494505
+        12           3   depth_grid_model_1   0.949084249084249
+        13           5   depth_grid_model_2  0.9484361792054099
+        14           1   depth_grid_model_0  0.9478162862778248
+
+    It appears that ``max_depth`` values of 9 to 27 are best suited for this dataset, which is unusally deep.
```

Comment:
Should this line be a comment? Looks like the indentation for this Python tab is being thrown off by this line
Minor typo ('unusually')